picard/ui/metadatabox.py

```diff
@@ -561,9 +561,24 @@ def update(self, drop_album_caches=False):
 
     def _update_tags(self, new_selection=True, drop_album_caches=False):
         self.selection_mutex.lock()
-        files = self.files
-        tracks = self.tracks
-        self.selection_mutex.unlock()
+        try:
```

Comment:
Why have you introduced a try block here? What exceptions are you expecting?

There is no `except` statement - so what happens if there is an exception in the `try` section?

Reply:
Thank you for pointing that out, I forgot to add the except block but I simply couldn't get it to work on my machine without using a try block. I'll look into it properly.